sunpy/data/_sample.py

```diff
@@ -116,6 +119,9 @@ def download_sample_data(overwrite=False):
 
         if not results.errors:
             return results
+        else:
+            log.info('Failed to download file' + str(err.filepath_partial) +
+                      'from url: ' + str(new_url) + ' with error: ' + str(err.exception))
```

Comment:
I wonder if this log info should be in the for loop below?

Reply:
Yes, that makes sense, added it below.